python/multi_align

```diff
@@ -276,7 +282,10 @@ if __name__ == '__main__':
     align_cmd = 'pyalign'
     for o, a in opts:
         if o in ("--tiers"):
-            tiernames = a.split(',')
+            tiersfull = [item + ':1'  if re.search(':[0-9]$',item) is None else item for item in a.split(',')] # default to channel 1
+            tiersdict = dict(item.split(':') for item in tiersfull) # convert to dict with tier names as keys and channel as values
```

Comment:
pat = re.compile(r'(.+):(\d+)$')
for o, a in opts:
...
    tiersfull = [item + ':1'  if pat.search(item) is None else item for item in a.split(',')]
   tiersdict = dict((m.group(1), m.group(2)) for i in tiersfull for m in [pat.search(i)])